### PR TITLE
Add pathing module and split preview

### PR DESCRIPTION
**Pull request by manuel-castro**
These features exist in other CG (eg fly). Adding them to Minnie here

### CODE REVIEW

File: pychunkedgraph/graph_analysis/analysis.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
+import fastremap
```
Comment (akhileshh) [inline pychunkedgraph/graph_analysis/analysis.py:1]: I feel this is better placed inside `graph`, maybe `graph.analysis.py` module or `graph.analysis` sub package if you expect to add more stuff overtime

File: pychunkedgraph/app/segmentation/common.py [hunk 2]
```diff
@@ -20,8 +20,8 @@
 from pychunkedgraph.app.meshing.common import _remeshing
 from pychunkedgraph.graph import exceptions as cg_exceptions
 # from pychunkedgraph.graph import history as cg_history
-from pychunkedgraph.graph import attributes
-# from pychunkedgraph.graph_analysis import analysis, contact_sites
+from pychunkedgraph.graph import attributes, cutting
```
Comment (akhileshh) [inline pychunkedgraph/app/segmentation/common.py:23]: better to keep each import in it's own line, reduces conflicts

File: pychunkedgraph/graph_analysis/analysis.py (hunk 1 above)
Comment (akhileshh) [reply pychunkedgraph/graph_analysis/analysis.py:1]: also keep analysis code together but move meshing code to `pychunkedgraph.meshing`

Comment (manuel-castro) [reply pychunkedgraph/graph_analysis/analysis.py]: I agree. I moved it to a new graph.analysis subpackage

**Comment by akhileshh** [comment]
if I'm not wrong, `graph.analysis` needs a `__init__.py` file?

**Pull request merged**